requirements.txt

```diff
@@ -83,4 +83,4 @@ wsgiref==0.1.2
 WTForms==2.0.2
 XlsxWriter==0.6.6
 xlrd==1.0.0
--e git+https://github.com/Code4SA/za-parliament-scrapers.git@master#egg=za_parliament_scrapers
```

Comment:
I'm not sure about this change - is this needed to make things work? It should work fine using the git dependency

Reply:
I changed the scrappers to origianal, but it is not working in my side

Reply:
I got the errors as before
![image](https://user-images.githubusercontent.com/33839033/62866199-e75ce380-bd18-11e9-9ca6-1a2cc6fe889d.png)

Reply:
We found that it works with the git dependency when we remove `-e`

We will keep it as a git dependency for now.